src/masoniteorm/schema/Blueprint.py

```diff
@@ -399,12 +406,17 @@ def __exit__(self, exc_type, exc_value, exc_traceback):
         return self.connection.query(self.to_sql(), ())
 
     def nullable(self):
-        """Sets the last column created as nullable
+        """Sets the last columns created as nullable
 
         Returns:
             self
         """
-        self._last_column.nullable()
+        _columns = []
```

Comment:
Here I deal with multiple locally in nullable() method but we could also have done it globally in the Blueprint class by  always using self._last_column as a list.
Don't know what's best